ocs_ci/utility/utils.py

```diff
@@ -4546,6 +4546,23 @@ def archive_ceph_crashes(toolbox_pod):
     toolbox_pod.exec_ceph_cmd("ceph crash archive-all")
 
 
+def ceph_crash_info_display(toolbox_pod):
+    """
+    Displays ceph crash information
+
+    Args:
+        toolbox_pod (obj): Ceph toolbox pod object
+
+    """
+    ceph_crashes = get_ceph_crashes(toolbox_pod)
+    for each_crash in ceph_crashes:
+        log.info(f"ceph crash: {each_crash}")
```

Comment:
Shouldn't we move here to warning or error level?

Reply:
done